code-challenges.test.js

```diff
@@ -22,13 +22,22 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
-
-
-
+// .map()
+const mixed_array = (array) => {
+  let color_array = []
+  for (var i = 0; i < array.length; i++) {
+    array[i].shift(" ")
```

Comment:
```suggestion
  let color_array = []
  array.shift()
  for (var i = 0; i < array.length; i++) {

```

`.shift()` only needs to be called once because it's actually removing the first element in the array. Because this line of code is getting called within the for-loop, it's going to remove all the elements in the array. The method `.shift()` also only is called on arrays (not an individual element within the array). `.shift()` also does not have any parameters so you do not need to provide `" "` as an argument